src/Command/InfectionCommand.php

```diff
@@ -171,7 +172,17 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $container = $this->getContainer();
-        $testFrameworkKey = $input->getOption('test-framework');
+
+        $bootstrap = $container->get('infection.config')->getBootstrap();
+        if ($bootstrap) {
+            if (!file_exists($bootstrap)) {
+                throw LocatorException::fileOrDirectoryDoesNotExist($bootstrap);
+            }
+
+            require_once $bootstrap;
+        }
+
+        $testFrameworkKey = $input->getOption('test-framework') ?? $container->get('infection.config')->getTestFramework();
```

Comment:
I believe `?:` instead of `??`

Reply:
I updated the default value to be null (aka fall back to config/default). ?? is more appropriate in that situation.